test/qa/lib/ssh.py

```diff
@@ -64,6 +67,50 @@ def __init__(self, ssh_class_instance):
         def get_file_hash(self, file_path: str) -> str:
             return self.ssh_class_instance.exec_command(f"{lib.FILE_HASH_UTILITY} {file_path}").split()[0]
 
+        def create_directory(self, file_count: int, name_suffix: str = "", parent_dir: str = "", file_size: str = "1K") -> Directory:
```

Comment:
Yes, that is correct. Did not want to complicate that function more, so decided to create a new one in `ssh.py` module. Although now that I am looking at it, maybe it would make sense to add `ssh_client` parameter to the function located inside of `fileshare.py`, and not rewrite pretty much same thing twice :thinking: